src/app/pages/ArticlePage/PagePromoSections/RelatedContentSection/Grid/index.jsx

```diff
@@ -0,0 +1,23 @@
+import Grid from '#app/components/Grid';
```

Comment:
We are moving away from using `psammead grid` in new code I think this should be implemented with `flex` layout or similar so it works in Opera Mini.